.github/workflows/ai-archaeology.yml

```
@@ -0,0 +1,342 @@
+            const safeToRemove = analysis.files.filter(f =>
+              f.current_relevance === 'deprecated' && f.removal_risk === 'low'
+            );
+
+            for (const file of safeToRemove.slice(0, 5)) {  // Limit to 5 issues
+              await github.rest.issues.create({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                title: `🏛️ Modernize: ${file.path}`,
+                body: `## Code Archaeology Findings
+
+          **File**: \`${file.path}\`
```

actionlint (1.7.8)

could not parse as YAML: could not find expected ':'

(syntax-check)